diff --git a/environments/dev/modules/sdkman/manifests/package.pp b/environments/dev/modules/sdkman/manifests/package.pp
index d7df367..83348ea 100644
--- a/environments/dev/modules/sdkman/manifests/package.pp
+++ b/environments/dev/modules/sdkman/manifests/package.pp
@@ -50,7 +50,7 @@ define sdkman::package (
   if $ensure == present and $is_default {
     exec {"sdk default $package_name $version" :
       environment => $sdkman::base_env,
-      command     => "su -c '$sdkman_init && sdk default $package_name $version' -${::sdkman::owner}",
+      command     => "su -c '$sdkman_init && sdk default $package_name $version' - ${::sdkman::owner}",
       user        => 'root',
       path        => '/usr/bin:/usr/sbin:/bin',
       logoutput   => true,
diff --git a/environments/dev/modules/vim/manifests/pathogen.pp b/environments/dev/modules/vim/manifests/pathogen.pp
index 53aac66..50eec0e 100644
--- a/environments/dev/modules/vim/manifests/pathogen.pp
+++ b/environments/dev/modules/vim/manifests/pathogen.pp
@@ -30,7 +30,7 @@ define vim::pathogen (
   exec { 'curl-pathogen':
     creates => "${home_real}/.vim/autoload/pathogen.vim",
     path    => ['/bin', '/usr/bin', '/usr/local/bin'],
-    command => "curl -LSso ${home_real}/.vim/autoload/pathogen.vim https://tpo.pe/pathogen.vim",
+    command => "curl -LSso ${home_real}/.vim/autoload/pathogen.vim https://raw.githubusercontent.com/tpope/vim-pathogen/v2.4/autoload/pathogen.vim",
     require => Package['curl'],
   }
 
diff --git a/environments/dev/modules/virtualbox/manifests/kernel.pp b/environments/dev/modules/virtualbox/manifests/kernel.pp
index 92c227c..911dde8 100644
--- a/environments/dev/modules/virtualbox/manifests/kernel.pp
+++ b/environments/dev/modules/virtualbox/manifests/kernel.pp
@@ -20,7 +20,7 @@ class virtualbox::kernel (
   exec { 'vboxdrv':
     command     => "${vboxdrv_command} setup",
     unless      => '/sbin/lsmod | grep vboxdrv',
-    environment => 'KERN_DIR=/usr/src/kernels/`uname -r`',
+    environment => 'KERN_VER=`uname -r`',
     require     => Package[$vboxdrv_dependencies],
   }
 
